.travis.yml

```diff
@@ -14,12 +14,14 @@ env:
   matrix:
   - MAVEN_VERSION=3.5.4 TEST=core
   - MAVEN_VERSION=3.5.4 TEST=python-validator
+  - MAVEN_VERSION=3.5.4 TEST=python-oncokb-import
+  - MAVEN_VERSION=3.5.4 TEST=python-oncokb-update
   - MAVEN_VERSION=3.5.4 TEST=sanity-checks
   - MAVEN_VERSION=3.5.4 TEST=end-to-end
   - MAVEN_VERSION=3.5.4 TEST=integration-test-load-study
 install:
 - |
-  if [[ "${TEST}" == core || "${TEST}" == end-to-end  || "${TEST}" == integration-test-load-study ]]
+  if [[ "${TEST}" == core || "${TEST}" == end-to-end  || "${TEST}" == integration-test-load-study || "${TEST}" == python-oncokb-update  ]]
```

Comment:
we've had some issues using Travis recently (it's very slow), would be better to use CircleCI or github actions in the future. Thanks for adding this test tho!

Reply:
Ok, thnx.